wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -833,6 +836,7 @@ extension ZMUserSession: ZMSyncStateDelegate {
 
             self.hasCompletedInitialSync = true
             self.notificationDispatcher.isEnabled = true
+            delegate?.clientCompletedInitialSync(accountId: account.userIdentifier)
             ZMUserSession.notifyInitialSyncCompleted(context: managedObjectContext)
```

Comment:
question: is this still needed ?

Reply:
Yes, there's other places in the app which still rely on this. Maybe that's stupid and we remove them but I didn't go down that rabbit hole :)